pkg/virt-controller/services/template.go

```diff
@@ -1143,6 +1161,101 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 	return &pod, nil
 }
 
+func (t *templateService) RenderHotplugAttachmentPodTemplate(volume *v1.Volume, ownerPod *k8sv1.Pod, vmi *v1.VirtualMachineInstance, pvcName string, isBlock bool) (*k8sv1.Pod, error) {
+	zero := int64(0)
+	pod := &k8sv1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "hp-volume-",
```

Comment:
can we put the vmi name in this GenreateName similar to how we do in virt-launcher so we'll be able to correlate disk pods with vmi pod